hugo/plan.sh

```diff
@@ -1,15 +1,15 @@
 pkg_name=hugo
 pkg_origin=core
-pkg_version=0.56.3
+pkg_version=0.57.0
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("Apache-2.0")
 pkg_description="Hugo is one of the most popular open-source static site generators."
 pkg_build_deps=(
   core/go
   core/git
-  core/gcc
 )
 pkg_deps=(
+  core/gcc
```

Comment:
Wait, what? How is this needed? (Genuinely curious)

Reply:
The extended version of hugo has a sass plugin that requires a c++ compiler.

Reply:
Ah I see. Thanks for explaining

Reply:
There's no way to build that at build time I suppose?

Reply:
No, sorry. Whatever this extension is it needs access to `libstdc++.so.6` at runtime:

```
[2][default:/src:0]# hab pkg exec tduffield/hugo hugo version
/hab/pkgs/tduffield/hugo/0.57.0/20190815141341/bin/hugo: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```

Reply:
I imagine this is part of the reason why this extension is not included in the core of hugo. Hugo, without this extension, has no external runtime dependencies.

Reply:
That being said, Hugo isn't as useful without this extension, and almost all distributions of hugo you'll get elsewhere are this extended version.

Reply:
Thanks for trying. I didn't mean these comments as blocking issues, just curious  😃

Reply:
No worries, I didn't see them as such. This was a good discussion to have for the historical context.

Reply:
If you want to avoid pulling in all of gcc as a runtime dep, in most cases you can depend on gcc-libs which includes things like libstdc++. Sometimes you'll need to to muck about with some flags to get it picked up correctly though.

Reply:
🤦‍♂ I see gcc-libs was already pointed out, apologies :D